### PR TITLE
net-irc/ngircd: bump to version 24

### PR DESCRIPTION
**Pull request by f1rebird**
- Updated copyright years
- Updated ebuild to EAPI6
- Cleaned up USE=ssl
- Added USE=test and dependencies to prepare for unrestricting
  FEATURES=test (one testcase is currently failing)

Signed-off-by: Moritz Kick <gentoo@blackphoenix.de>

### CODE REVIEW

**Comment by f1rebird** [comment]
@gentoo/proxy-maint

File: net-irc/ngircd/ngircd-23-r1.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (SoapGentoo) [inline net-irc/ngircd/ngircd-23-r1.ebuild]: why did you update this file?

File: net-irc/ngircd/ngircd-24.ebuild [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+src_install() {
+	default
+	newinitd "${FILESDIR}"/ngircd.init-r1.d ngircd
+}
+
+pkg_postinst() {
+	if ! use prefix; then
+		enewuser ngircd
+		chown ngircd "${ROOT}"/etc/ngircd/ngircd.conf || die
```
Comment (SoapGentoo) [inline net-irc/ngircd/ngircd-24.ebuild]: `chown ngircd "${EROOT%/}"/etc/ngircd/ngircd.conf || die`

File: net-irc/ngircd/ngircd-24.ebuild [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+IUSE="debug gnutls iconv ident ipv6 libressl pam ssl tcpd test zlib"
+
+RDEPEND="
+	iconv? ( virtual/libiconv )
+	ident? ( net-libs/libident )
+	pam? ( virtual/pam )
+	ssl? (
+		gnutls? ( net-libs/gnutls )
```
Comment (SoapGentoo) [inline net-irc/ngircd/ngircd-24.ebuild]: needs `:=`

File: net-irc/ngircd/ngircd-23-r1.ebuild (hunk 1 above)
Comment (f1rebird) [inline net-irc/ngircd/ngircd-23-r1.ebuild]: was a mistake, reverted

File: net-irc/ngircd/ngircd-24.ebuild (hunk 2 above)
Comment (f1rebird) [inline net-irc/ngircd/ngircd-24.ebuild]: fixed.

File: net-irc/ngircd/ngircd-24.ebuild (hunk 3 above)
Comment (f1rebird) [inline net-irc/ngircd/ngircd-24.ebuild]: Just gnutls, or pam and libident also?

**Comment by f1rebird** [comment]
Requested changes applied

**Comment by f1rebird** [comment]
@gentoo/proxy-maint
I applied all requested changes, any news about this or am I still missing something?

**Comment by kensington** [comment]
I apologise for the delay in attending to this PR. I have picked it as 0f6ea10975d4557d8f5203922daad9c05bd914dd, dropping the copyright year change to ``ngircd.init-r1.d`` as the file didn't see any changes.

**Pull request closed without merging**